filesystem/fat32/fat32.go

```diff
@@ -228,6 +228,17 @@ func Create(f util.File, size int64, start int64, blocksize int64, volumeLabel s
 		return nil, fmt.Errorf("Wrote %d bytes of MS-DOS Boot Sector to disk instead of expected %d", count, SectorSize512)
 	}
 
+	// write backup to the file
+	if fsisBackupSector > 0 {
```

Comment:
I don't get this. Why are we writing the boot sector bytes to the FSIS Backup Sector?

Reply:
And then we write it again later?